virttest/qemu_vm.py

```diff
@@ -1055,6 +1055,14 @@ def add_memorys(devices, params):
                 backend_options["size_mem"] = "%sM" % params["mem"]
                 if params.get("vm_mem_backend"):
                     backend_options["backend_mem"] = params.get("vm_mem_backend")
+                if params.get("vm_mem_backend") == "memory-backend-file":
+                    if not params.get("vm_mem_backend_path") == "":
+                        raise virt_vm.VMConfigMissingError(name,
+                                                           'vm_mem_backend_path'
+                                                           'is missing')
```

Comment:
No need `is missing`,  the exception `VMConfigMissingError` will handle it automatically:
https://github.com/avocado-framework/avocado-vt/blob/e20d6f47c2198f423bbc8f420b2775e404322416/virttest/virt_vm.py#L93

Reply:
updated